### DISCUSSION TITLE
What's the difference between yellow and green blocks in Modern theme?

### DISCUSSION THREAD

**Discussion by endolith** [WinMerge/winmerge#2226] [Q&A] [answered]
Example:

![image](https://github.com/WinMerge/winmerge/assets/58611/d9427053-5603-488b-a978-b9c6db3bca97)

The yellow blocks are not as useful because they don't highlight the actual differences (in this case the capital `H`).  I would like to see the green differences on every line, not the yellow.

**Comment by endolith** [comment]
Oh I see in the settings, yellow are "ignored differences", and green are "differences":

![image](https://github.com/WinMerge/winmerge/assets/58611/8c6420c5-7038-4beb-9dce-15a2e1bad2dc)

So how do I "not ignore" all differences?  I don't have anything ignored:

![image](https://github.com/WinMerge/winmerge/assets/58611/6f17d708-ad8e-4f38-8409-29681f6fefcc)

**Reply by endolith** [reply]
It looks as though numbers and case differences are "ignored" whether I check those boxes or not?

**Comment by sdottaka** [comment, marked as answer]
Isn't IgnoreCommentsC displayed in the status bar as shown below? This is a Prediffer plugin to ignore C language comments.


![image](https://github.com/WinMerge/winmerge/assets/98126/82ce16d4-c754-46f0-9b87-4f595edc7a56)


If you have not applied it intentionally, Plugins → Automatic Prediffer menu item is probably selected, so select Plugins → Manual Prediffer menu item to change it back.


![image](https://github.com/WinMerge/winmerge/assets/98126/118a3549-9384-4c2a-8c0b-06d9c456e0de)
